libs/cardano-ledger-api/test/Test/Cardano/Ledger/Api/Tx/Body.hs

```diff
@@ -141,7 +219,5 @@ spec =
       prop "evalBalanceTxBody" $ propEvalBalanceTxBody @Alonzo
     describe "BabbageEra" $ do
       prop "evalBalanceTxBody" $ propEvalBalanceTxBody @Babbage
-
--- TODO: bring back when TxCerts become a type family
--- describe "ConwayEra" $ do
---   prop "evalBalanceTxBody" $ propEvalBalanceTxBody @Conway
+    describe "ConwayEra" $ do
+      prop "evalBalanceTxBodyConway" $ propEvalBalanceTxBodyConway @Conway
```

Comment:
Get rid of `evalBalanceTxBodyConway` and `propEvalBalanceTxBodyConway`. Copy pasting functionality in order to test that functionality matches the copy is not a test.

All you needed to do was uncomment the piece of code with a TODO

```suggestion
      prop "evalBalanceTxBody" $ propEvalBalanceTxBody @Conway
```